plugin.rb

```diff
@@ -84,5 +85,37 @@
         end
       end
     end
+
+    add_automation_scriptable("send_chat_integration_message") do
+      field :provider,
+            component: :choices,
+            extra: {
+              content:
+                DiscourseChatIntegration::Provider.enabled_provider_names.map do |provider|
+                  { id: provider, name: "chat_integration.provider.#{provider}.title" }
+                end,
+            },
+            required: true
+      field :channel_name, component: :text, required: true
+
+      version 1
+
+      triggerables %i[topic_tags_changed]
+
+      script do |context, fields, automation|
+        provider = fields.dig("provider", "value")
+        channel_name = fields.dig("channel_name", "value")
+
+        post = DiscourseChatIntegration::ChatIntegrationReferencePost.new(context)
+        provider = DiscourseChatIntegration::Provider.get_by_name(provider)
+
+        channel = provider.get_channel_by_name(channel_name) # user must have created a channel in /admin/plugins/chat-integration/<provider> page
```

Comment:
Should we capture a specific exception in this case and log a specific message to explain this?